docs/docs/atom-archive/getting-started/sections/installing-atom.md

```diff
@@ -0,0 +1,264 @@
+
+{{#linux}}
+
+![Download buttons on https://atom.io](../../images/linux-downloads.png "Download buttons on https://atom.io")
+
+{{/linux}}
+
+The button or buttons should be specific to your platform and the download package should be easily installable. However, let's go over them here in a bit of detail.
+
+{{#mac}}
+
+#### Installing Atom on Mac
+
+Atom follows the standard Mac zip installation process. You can either press the download button from the https://atom.io site or you can go to the [Atom releases page](https://github.com/atom/atom/releases/latest) to download the `atom-mac.zip` file explicitly. Once you have that file, you can click on it to extract the application and then drag the new `Atom` application into your "Applications" folder.
+
+When you first open Atom, it will try to install the `atom` and `apm` commands for use in the terminal. In some cases, Atom might not be able to install these commands because it needs an administrator password. To check if Atom was able to install the `atom` command, for example, open a terminal window and type `which atom`. If the `atom` command has been installed, you'll see something like this:
+
+``` command-line
+$ which atom
+> /usr/local/bin/atom
+$
+```
+
+If the `atom` command wasn't installed, the `which` command won't return anything:
+
+``` command-line
+$ which atom
+$
+```
+
+To install the `atom` and `apm` commands, run "Window: Install Shell Commands" from the [Command Palette](/getting-started/sections/atom-basics#command-palette), which will prompt you for an administrator password.
+
+{{/mac}}
+
+{{#windows}}
+
+#### Installing Atom on Windows
+
+Atom is available with Windows installers that can be downloaded from https://atom.io or from the [Atom releases page](https://github.com/atom/atom/releases/latest).  Use `AtomSetup.exe` for 32-bit systems and `AtomSetup-x64.exe` for 64-bit systems. This setup program will install Atom, add the `atom` and `apm` commands to your `PATH`, and create shortcuts on the desktop and in the start menu.
+
+![Atom on Windows](../../images/windows-system-settings.png)
+
+The context menu `Open with Atom` in File Explorer, and the option to make Atom available for file association using `Open with...`, is controlled by the System Settings panel as seen above.
+
+With Atom open, click on `File > Settings`, and then the `System` tab on the left. Check the boxes next to `Show in file context menus`, as well as `Show in folder context menus`. And you’re all set.
+
+{{/windows}}
+
+{{#linux}}
+
+#### Installing Atom on Linux
+
+You can install Atom on Linux using your distribution's package manager by configuring it to use one of our official package repositories. This will also enable you to update Atom when new releases are published.
+
+##### Debian and Ubuntu (deb/apt)
+
+To install Atom on Debian, Ubuntu, or related distributions, add our official
+package repository to your system by running the following commands:
+
+``` command-line
```

Comment:
I might be outdated at this point, but it was what they originally had. I need to replace and fix this. But this is outdated at this point because of my most recent commit.